### PR TITLE
Set data changes description from bridge

### PR DESCRIPTION
**Pull request by kostasjurasbentley**

### CODE REVIEW

**Comment by calebmshafer** [comment]
> Functionality needed for https://dev.azure.com/bentleycs/iModelTechnologies/_workitems/edit/447508/

@kostasjurasbentley, can you avoid linking to internal only work items for descriptions and copy the relevant content into the description (without mentioning individual users)?

**Comment by calebmshafer** [comment]
@kostasjurasbentley working on a fix to the pipeline now. You're the first one to create a fork-based PR so a few things need to be tweaked first.

**Comment by calebmshafer** [comment]
@kostasjurasbentley just pushed a change to `master` that should fix the issue. Mind updating your fork?

**Comment by kostasjurasbentley** [comment]
> @kostasjurasbentley just pushed a change to `master` that should fix the issue. Mind updating your fork?

done

**Review by pmconne — APPROVED** [review]
Approving for /common/.

File: common/api/imodel-bridge.api.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ export abstract class IModelBridge {
     abstract getApplicationVersion(): string;
     // (undocumented)
     abstract getBridgeName(): string;
+    // (undocumented)
```
Comment (abeesh) [inline common/api/imodel-bridge.api.md:56]: Can you please add some documentation?

File: core/imodel-bridge/src/BridgeRunner.ts [hunk 2]
```diff
@@ -424,7 +424,11 @@ class BriefcaseDbBuilder extends IModelDbBuilder {
     // WIP: need detectSpatialDataTransformChanged check?
     await this._bridge.updateExistingData();
 
-    return this._saveAndPushChanges("Data changes", ChangesType.Regular);
+    let dataChangesDescription = "Data changes";
```
Comment (abeesh) [inline core/imodel-bridge/src/BridgeRunner.ts:427]: Can you check whether imodel.js have any precedence on string localization?

**Review by abeesh — APPROVED** [review]

**Comment by calebmshafer** [comment]
@kostasjurasbentley can you sign the CLA? After that we should be able to get this merged. Thanks!

**Comment by kostasjurasbentley** [comment]
> @kostasjurasbentley can you sign the CLA? After that we should be able to get this merged. Thanks!

done

**Pull request merged**